packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -471,6 +478,14 @@ abstract class ThirdPartyGitSource extends GitSource {
  */
 export interface CodeCommitSourceProps extends GitSourceProps {
   readonly repository: codecommit.IRepository;
+
+  /**
+   *  The commit ID, branch, or Git tag to use.
+   *
+   * @example 'mybranch'
+   * @default the default branch's HEAD commit ID is used
+   */
+  readonly sourceVersion?: string;
```

Comment:
I actually don't like this name ;(. In my opinion, 99% of the time, this will be a branch.

What do you think about the name `branchOrRef`?

Reply:
👍  I like that name.